### PR TITLE
Implemented CdnDTO for manifest import/export.

### PR DESCRIPTION
**Pull request by nikosmoum**
- CdnDTO is used instead of Cdn when importing/exporting a manifest.
- Refactored Importer.populateEntity() for certificates, as a general purpose populating method in the sync package for all certificate entities that extend RevocableCertificate.

### CODE REVIEW

File: server/src/main/java/org/candlepin/sync/Importer.java [hunk 1]
```diff
@@ -717,7 +723,7 @@ protected ConsumerDTO importConsumer(Owner owner, File consumerFile, File[] upst
      * @throws IllegalArgumentException
      *  if either entity or dto are null
      */
-    private void populateEntity(IdentityCertificate entity, CertificateDTO dto) {
+    static void populateEntity(RevocableCertificate entity, CertificateDTO dto) {
```
[the remark this answers is not in the document]
Comment (nikosmoum) [reply server/src/main/java/org/candlepin/sync/Importer.java]: I've moved this method to a ImporterUtils class for now, but I agree we should introduce some kind of EntityPopulator framework. Maybe something like `populate(entity, dto, strategy)` where `strategy` is a function that decides the context of how we populate (manifest/rules/api, if it is create/update, etc.)

**Comment by Ceiu** [comment]
ack

**Comment by nikosmoum** [comment]
@Ceiu I just realized something that I have doubts about. I changed the populateEntity method for certificates to not populate the `id` and `serial` fields of the CertificateSerial sub-entity because they are the same, and also using the default non-arg constructor which autogenerates the id when populating. But I'm not sure if that is right now. Maybe we should populate this value on the CertificateSerial entity from the DTO? And if we should, which of the two should we choose? (id and serial are different fields on the DTO, but the same on the entity).

**Comment by Ceiu** [comment]
The clearing of the ID is fine, and probably preferred so we don't run into collisions when saving. At best, I would use the IDs just to line things up before persisting, but still let new IDs be generated for imported items.

Regarding the sub-entity serial field, that we happen to know the ID is the same internally is a black-box detail -- one I'm only slightly worried about (due to the afore mentioned collision potential). The way you've written it should be fine and is how I would have done it myself.

**Pull request merged**